### DISCUSSION TITLE
3.0.0 Well Tested

### DISCUSSION THREAD

**Discussion by micsthepick** [micsthepick/JSVocalRedIso#13] [Announcements]
With the advance of automated tests, I thought it prudent to create a new release.

Preliminary listening seems to make it seem as if this is the best performing release so far,

Note that the sliders for phase width are now exponential, so they range from -5 to 5 now, not unlike the strength

There should be less popping frequency artifacts :) 🥇 

<hr /><em>This discussion was created from the release <a href='https://github.com/micsthepick/JSVocalRedIso/releases/tag/v3.0.0'>3.0.0 Well Tested</a>.</em>

**Comment by micsthepick** [comment]
Any praise or gripes from this new release?

let loose here!

**Reply by jamesWalker55** [reply]
![image](https://github.com/micsthepick/JSVocalRedIso/assets/30405220/0d6a5c82-1af4-47d7-8791-17af248e4e93)

I tried the eel version, on Reaper 7.11 it complains about a syntax error and has no audio output.

![image](https://github.com/micsthepick/JSVocalRedIso/assets/30405220/8b74eeff-c90b-4397-b417-d489c1eb272a)

I also tried the jsfx version. No syntax errors but it produces no output either.

**Reply by micsthepick** [reply]
ok, I'll look in to that, the eel is meant only for JamesDSP, but I had forgotten to test the JSFX version in reaper!

**Reply by micsthepick** [reply]
apparently freembuf doesn't like having 0 passed in, will have to make sure that it works correctly still further as my M1 ran fine, but the x86_64 version seemed to have nan when peeking at some vars? (might just be because of the freembuf issue).
